Userland/Applications/SoundPlayer/Player.cpp

```diff
@@ -155,3 +156,11 @@ void Player::seek(int sample)
 {
     m_playback_manager.seek(sample);
 }
+
+Vector<Audio::PictureData> Player::pictures() const
+{
+    auto* const plugin_ptr = m_playback_manager.loader()->plugin().ptr();
+    if (is<Audio::FlacLoaderPlugin>(plugin_ptr))
+        return ((Audio::FlacLoaderPlugin*)plugin_ptr)->pictures();
```

Comment:
Let's make that a virtual function for all audio loaders. (I have plans for a loader-wide metadata system anyways, so that will come in handy.)